themes/openy_themes/openy_carnation/templates/paragraph/paragraph--grid-content--default.html.twig

```diff
@@ -45,12 +45,20 @@ set classes = [
   view_mode ? 'paragraph--view-mode--' ~ view_mode|clean_class,
 ]
 %}
+{% set grid_style = content.field_prgf_grid_style[0]['#markup'] %}
+{% if grid_style == '2' %}
+  {% set item_class = 'col-12 col-sm-6 row-eq-height' %}
+{% elseif grid_style == '3' %}
+  {% set item_class = 'col-12 col-sm-4 row-eq-height' %}
+{% elseif grid_style == '4' %}
+  {% set item_class = 'col-12 col-sm-3 row-eq-height' %}
+{% endif %}
 <div{{ attributes.addClass(classes) }}>
   {% for key, item in content.field_grid_columns %}
     {% if key matches '/^\\d+$/' %}
-    <div class="col-12 col-lg">
-      {{ item }}
-    </div>
+      <div class="{{ item_class }}">
```

Comment:
what if grid_style not in (2,3,4) ?
```class=""``` breaks css validation rules

Reply:
hmm, at this moment we have only these options. But I agree, let's add a default value.